test/test-plugins.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
```

Comment:
It was useful to see the test instrumentation along side the api but I would prefer holding off on submitting these test files as they should only be need while both patching mechanisms are in place.